tests/unit/test_model.py

```diff
@@ -115,6 +115,31 @@ def setUp(self):
         }
         self.service_model = model.ServiceModel(self.model)
 
+    def test_api_name(self):
```

Comment:
you may want to rename these tests since that we use ``wire_name`` and ``name`` now.